src/modules/menu/components/activeLayers/MenuActiveLayersListItem.vue

```diff
@@ -178,6 +179,12 @@ export default {
             }
             return 'square'
         },
+        showLegendIcon() {
+            if (this.layer !== null) {
+                return this.layer.hasLegend
+            }
+            return false
```

Comment:
just for curiosity, you can "one line" this in JS with the optional chaining :
`return this.layer?.hasLegend`

with this notation, `hasLegend` will only be accessed if `this.layer` is valid (not null or undefined), otherwise it returns `null` (and as `null` is somewhat equivalent to `false` in JS, that works pretty well)